doc/developers/performance.rst

```diff
@@ -386,10 +386,17 @@ Checkout the official joblib documentation:
 - https://pythonhosted.org/joblib
 
 
-.. _warm-restarts:
+.. _warm-starts:
 
-A sample algorithmic trick: warm restarts for cross validation
+A sample algorithmic trick: warm starts for cross validation
 ==============================================================
 
-TODO: demonstrate the warm restart tricks for cross validation of linear
-regression with Coordinate Descent.
+One of the most common tasks when doing model selection is
+ref:`cross-validation <cross_validation>`. Since it is computationally
```

Comment:
I was confused by the section's title "warm restarts for cross validation". sorry for that